holoviews/element/path.py

```diff
@@ -148,7 +149,11 @@ def clone(self, *args, **overrides):
         settings = dict(self.get_param_values(), **overrides)
         if not args:
             settings['plot_id'] = self._plot_id
-        return self.__class__(*args, **settings)
+
+        pos_args = getattr(self, '_' + type(self).__name__ + '__pos_params', [])
```

Comment:
I believe single underscore would have messed up ``Layouts`` and ``AttrTrees``. It was done like this for a reason but I can't quite remember all the details.